sources/packages/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -87,10 +92,8 @@ export class RestrictionController extends BaseController {
    * REST API to provide the list of restriction categories for drop-down.
```

Comment:
Please remove the "REST API" from the beginning of all methods. IMO it is not adding value.